### ISSUE TITLE
problem with RTM tip

### ISSUE THREAD

**Issue by messschuh** [pavelrevak/rt_soldering_pen#9]
i use the v.03 HW since round 3 years most time with RT8MS tip, now i got a new one, the RTM032S and they work a bit strange.
the temperature is not stable(+/- 10°) like the RT8MS (+/- 3°) and the powerregulation seems diffrent. at freeair the RT8MS power is nearly stable at round 3W, with the RTM032S  the power alternates from 0W to 6W to 0W @ round 3 seconds and with this tip the pen never detect an idle and because of this never goes to idle mode.
 the heat resitance shown at oled is for RTM032S 1.8Ohm and for RT8MS 2.1Ohm.  i testes SW v0.4.0 and v0.3.0 without diffrents. 
only part of code i changed was the heating power, i reduced it to 35W because of 2 burned tips in the last 2 years.
i dont think this is normal, is this a known issue of this tip? can i try something?

**Comment by swissfreek** [comment]
A friend of mine has also had three or four tips burn up over the past year. The issues I’ve identified with him is that he runs it at 400° Or more all the time to compensate for the (relatively) lower power compared to like an 80W soldering station. I never run mine higher than 400° and haven’t had a problem. I suspect these tips don’t like being run at high temp for long periods of time

**Comment by pavelrevak** [comment]
RTM tips are very small, there is hard to tune PID regulation loop for all range of tips, for this tiny tip you can try to decrease amplification of all constants in PID (heating.hpp: PID_K_...)

**Comment by messschuh** [comment]
thanks for fast reply.
most time i use mine at a range of 330°-360°C, for harder soldering jobs i use my 80W station. i´m not really sure constant high temperature  was problem of burning my tips, i will try it with the reduced Power (35W) and will see what happens. 

what did u mean with "this tiny tip" ? the tips at all or especialy the RTM032S, because the RTM032S is a bit bigger than the RT8MS. 
Ok, i tried some diffrent PID vlaues and now i have setting that works for me. I decresse P and I to 350, 300 and increase D to 75. now it needs a bit more time to get stable temperature but the settings works with my old and the new tip. maybe some finetuning is needed to get it works perfect, but for the moment it is ok.

**Comment by pavelrevak** [comment]
Ah, I'm sorry, RTM032 is bigger, I was wrong with another RTP, which is smaller. but yes, it can help to tune the PID constants.

it will be good idea to make these constants editable from menu for fine-tune

**Comment by messschuh** [comment]
yes  thats a good idea, maybe another good idea is to implement a PID autotune function. i know PID autotune from 3d-printers and that works well. but i dont know it  would pe possible to implement it.

**Comment by messschuh** [comment]
short reply after nearly 2 years, no burned tips again and the "new" PID-settings works well. just at "higher" voltage ( 4s- range) the PID´s do not work perfect but acceptable good.
